### DISCUSSION TITLE
Common Alias tip and speed up Pikaur tip

### DISCUSSION THREAD

**Discussion by cg00001** [actionless/pikaur#708] [Show and tell]
I was lucky enough to discover Pikaur the same year it was out.
I think almost everyone uses Aliases or related commands, these are the ones I've been using with Pikaur -mostly-, on Fish.
```
# Pikaur basic aliases
alias upd="pikaur -Syyu"
alias pi="pikaur -Syu"
alias pir="pikaur -R"
alias pis="pikaur -Ss"
alias piq="pikaur -Qs"
alias pii="pikaur -Si"
alias pid="pikaur -Sii"
alias paca="pikaur -Scc"
# Remove orphans
alias po="sudo pacman -Qdtq | sudo pacman -Rns -"
# Search file inside packages
alias pf="pkgfile -s"
alias pf2="sudo pacman -Fy"
# Remove package even though dependencies don't let it
alias pd="sudo pacman -Rd --nodeps"
```

To see the few ways I made Pikaur even faster, here's my pikaur.conf
https://bin.disroot.org/?dd0ef66c09e213b0#22rKeCteA1fAp7d6qHfPN2uwWgsivRv7DikiHR9nv1J1

**Comment by actionless** [comment]
i personally have 2 aliases:

```
p=pikaur
syu=pikaur -Syu
```

and script similar to your:

> po="sudo pacman -Qdtq | sudo pacman -Rns -"
